test/test_helper.rb

```diff
@@ -6,7 +6,7 @@
 class ActiveSupport::TestCase
   # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
   DatabaseCleaner.strategy = :transaction
-  SimpleCov.start
+  SimpleCov.start('rails')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.